templates/document_head.hbs

```diff
@@ -16,14 +16,11 @@
 <script type="module">
     import { setupGardenTheme } from "shared";
 
-    setupGardenTheme(function (defaultTheme) {
-        return {
-            ...defaultTheme,
-            colors: {
-                ...defaultTheme.colors,
-                foreground: "{{settings.text_color}}",
-                primaryHue: "{{settings.brand_color}}",
-            },
-        }
```

Comment:
Is there an advantage for moving this logic to `shared.tsx`?

I am thinking that, since this template is editable in the theme editor, it would make it easier / straightforward to change the styling if we build the Garden theme here. What do you think?

Reply:
It makes sense, I moved it to the `shared.tsx` because it is possible to import things from NPM modules, we especially need the `css` function from `styled-components`, but we can eventually re-export it from `shared.tsx` if we want to.

Let's merge this PR and eventually revisit it in another PR